lib/gears/timer.lua

```diff
@@ -145,7 +145,7 @@ end
 -- @see start
 -- @see stop
 
---- The timer timeout value.
+--- The timer timeout value in seconds.
```

Comment:
this i like 👍
but other changes look like just replacing one synonymous term to other 🤷

Reply:
I think specifying that the "countdown" "immediately" starts, would be more clear than stating that the timer "automatically starts".